packages/admin/admin/src/form/fields/SelectField.tsx

```diff
@@ -27,10 +27,12 @@ export interface SelectFieldProps<Value extends string | number> extends SelectF
 
 export function SelectField<Value extends string | number>({ componentsProps = {}, children, options, ...restProps }: SelectFieldProps<Value>) {
     const { finalFormSelect: finalFormSelectProps } = componentsProps;
+    const { startAdornment, endAdornment } = restProps;
+
     return (
         <Field {...restProps}>
             {(props) => (
-                <FinalFormSelect<Value> {...props} {...finalFormSelectProps}>
+                <FinalFormSelect<Value> {...props} {...finalFormSelectProps} startAdornment={startAdornment} endAdornment={endAdornment}>
```

Comment:
Is this change necessary? Didn't this already work before? startAdornment and endAdornment should be in ...props